dist/components/toast.css

```diff
@@ -60,19 +61,45 @@
   padding: 0.78571429em 1em;
   margin: 0 0 6px;
   color: rgba(255, 255, 255, 0.9);
-  opacity: 0.8;
+  opacity: 0.9;
+}
+
+.floating.toast {
+  box-shadow: 0 0 0 1px #a3c293 inset, 0 2px 4px 0 rgba(34,36,38,.12), 0 2px 10px 0 rgba(34,36,38,.15);
```

Comment:
I really like this ! Since we got rid of opacity, what do you think about display it (or a variant) on mouse over ?

Reply:
I suggest implementing this as an additional class, so the user can decide how it should behave

Will only show the box-shadow on hover
```css
className: {
	toast: 'ui hoverfloating toast'
}
```

Will still show the box shadow any time
```css
className: {
	toast: 'ui floating toast'
}
```

Reply:
Changed / added accordingly by commit f49a63cf3100149835360a7ecaef4e6d4c07b9ee.